docker/debian/Dockerfile

```diff
@@ -31,10 +31,12 @@ RUN install_packages avahi-daemon ca-certificates docker.io jq openssh-client
 
 COPY docker/docker-init.sh init.sh
 
-RUN chmod +x ./init.sh
+RUN CLI_CMDS=$(jq -r '.commands | keys | map(.[0:index(":")]) | unique | join("\\ ")' < oclif.manifest.json); \
+    sed -ie "s/CLI_CMDS=\"help\"/CLI_CMDS=\"help\\ ${CLI_CMDS}\"/" init.sh && \
+    chmod +x init.sh
```

Comment:
Yes, the expressions are a bit ugly. :-) But only 2 lines of that, and it seems to work.